etc/adminhtml/system.xml

```diff
@@ -45,11 +45,27 @@
                     <label>Webhook Secret</label>
```

Comment:
@ramth05 Can you make it required filed or show some message if merchant leaves this field as blank in case if he had enabled the webhook!

Reply:
done